docs/_docs/best-practices/general/general.md

```diff
@@ -28,6 +29,157 @@ Discover essential FinOps best practices to optimize cost efficiency and governa
 
 ---
 
+## Resource inventory summary
+
+### Query: Count of all resources
+
+This Azure Resource Graph (ARG) query counts all resources within the specified scope.
+
+<h4>Category</h4>
+
+Inventory
+
+<h4>Query</h4>
+
+```kql
+Resources
+| summarize count()
+```
+
+<br>
+
+### Query: Count of all resources per subscription
+
+This Azure Resource Graph (ARG) query counts all resources within the specified scope per subscription.
+
+<h4>Category</h4>
+
+Inventory
+
+<h4>Query</h4>
+
+```kql
+resources
+| summarize Count=count(id) by subscriptionId
+| order by Count desc
+```
+
+<br>
+
+### Query: Count of all resources per type
+
+This Azure Resource Graph (ARG) query counts all resources within the specified scope per type.
+
+<h4>Category</h4>
+
+Inventory
+
+<h4>Query</h4>
+
+```kql
+Resources 
+| extend type = case(
+type contains 'microsoft.netapp/netappaccounts', 'NetApp Accounts',
```

Comment:
This is unmaintainable and not technically accurate, if I'm reading it correctly. We should discuss this one.